### ISSUE TITLE
pip v19.0 removed --process-dependency-links

### ISSUE THREAD

**Issue by yamikaitou** [Cog-Creators/Red-DiscordBot#2385]
# Other bugs

#### What were you trying to do?

Install Red into a pyenv venv

#### What were you expecting to happen?

Red to install as normal

#### What actually happened?

```pi@terraform:~$ pyenv virtualenv 3.7.0 red
Using base prefix '/home/pi/.pyenv/versions/3.7.0'
New python executable in /home/pi/.pyenv/versions/3.7.0/envs/red/bin/python3.7
Also creating executable in /home/pi/.pyenv/versions/3.7.0/envs/red/bin/python
Installing setuptools, pip, wheel...done.
Looking in links: /tmp/tmp4kehpb3t
Requirement already satisfied: setuptools in /home/pi/.pyenv/versions/3.7.0/envs/red/lib/python3.7/site-packages (40.6.3)
Requirement already satisfied: pip in /home/pi/.pyenv/versions/3.7.0/envs/red/lib/python3.7/site-packages (19.0)
pi@terraform:~$ pyenv shell red
(red) pi@terraform:~$ python -V
Python 3.7.0
(red) pi@terraform:~$ python -m pip install -U --process-dependency-links --no-cache-dir Red-DiscordBot

Usage:   
  /home/pi/.pyenv/versions/red/bin/python -m pip install [options] <requirement specifier> [package-index-options] ...
  /home/pi/.pyenv/versions/red/bin/python -m pip install [options] -r <requirements file> [package-index-options] ...
  /home/pi/.pyenv/versions/red/bin/python -m pip install [options] [-e] <vcs project url> ...
  /home/pi/.pyenv/versions/red/bin/python -m pip install [options] [-e] <local project path> ...
  /home/pi/.pyenv/versions/red/bin/python -m pip install [options] <archive url/path> ...

no such option: --process-dependency-links
(red) pi@terraform:~$ 
```

#### How can we reproduce this issue?

Upgrade pip to v19.0 and install Red

**Comment by lionirdeadman** [comment]
This will be fixed with #2387

**Comment by Tobotimus** [comment]
Resolved by #2387.
